microhab/SPEXpdd.tex

```diff
@@ -0,0 +1,253 @@
+project shall be designed to be engineered in a fashion that teaches both
+established senior members and new greenhorn SPEX members alike.
+
+Electrical hardware design is a skill that many members have shown a desire to
+pick up. Design of the PCB shall be simplistic and incorporate a few different
+important skills for engineering in the electrical hardware domain.
+
+Embedded software is another useful skill that many students in the Computing
+skill aren't taught formally. Arduino development and embedded C software
+engineering skills will be obtained from writing code for the avionics board.
+
+\section{Benefit to SPEX}
+\label{sec:benefit}
+% One of the core values of SPEX is to provide opportunities for academic and professional growth for its members,
+% and to challenge them with interesting projects.
+% In this section, explain how the project would benefit SPEX members as students,
+% space enthusiasts, and young professionals.
+
+Each of the objectives grant their own benefit to SPEX.\@ Cheaper launch costs
+allow for more SPEX funding to be allocated to other projects while not
+sacrificing the quality of HAB launches significantly.
+
+Educational benefit, while not the primary benefit, could be the largest
+benefit.  PCB design has always been a very in demand with low supply skill.
+Embedded software is required for almost every large project. Arduino software
+design is a great stepping stone for software oriented people into the world of
+embedded. Avionics design in C would be the final transition, from Arduino into
+the real world application of embedded software.
+
+% Below I have used subsections to identify key ideas in this section. These particular subsections are not required as part of the SPEX Standard, but serve as an example of using subsections in a text.
+
+\subsection{Accessibility}
+\label{subsec:plug-n-play}
+  % Note below that LaTeX uses weird formatting when it comes to quotation marks.
+  % The style below is correct to display forward quotes `` at the start of the phrase and backquotes '' at the end.
+
+In regards to the software half, Arduino is a very approachable language.  There
+is a vast amount of libraries out there already for interfacing with ancillary
+hardware such as sensors or memory. The C portion to come at a later date (or in
+parallel) will be quite a bit more difficult; it requires a ton of datasheet
+reading, innovative code (not taking from examples), and customizing toolchains.
+
+PCB design accessibility is a lot more restricted. Members that have had PCB
+design classes (4th-5th year Electrical Engineers) or have experience from co-op
+will be able to contribute to the design.
+
+
+
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+Documentation for each step of the respected engineering paths is required. It
+is important during the implementation aspect to retain problems experienced and
+the solutions used to overcome.
```

Comment:
more?

What are some design tenets that you think microHabs should adhere to?

What is the cutoff point for a microhab to just be a regular hab?

Will the microhab design process differ from a regular hab process?

will each microhab be different or part of a common design?